hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -279,6 +283,7 @@ public synchronized void close() throws IOException {
         threadExecutor.shutdownNow();
       }
     }
+    LOG.debug("Closing AbfsOutputStream ", toString());
```

Comment:
let's guard this with a LOG.isDebugEnabled(), because that toString() operation is doing enough work. Or, use `this` as the argument and have SLF4J Call this.toString() only if it is printing the log entry